src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -200,15 +200,18 @@ private List<String> validateEmailContactAndDomain(UserRegistration userRegistra
         if (!ProjectUtil.validateFirstName(userRegistration.getFirstName())) {
             errList.add("Invalid First Name");
         }
-        if (!ProjectUtil.validateLastName(userRegistration.getLastName())) {
-            errList.add("Invalid Last Name");
-        }
         if (!ProjectUtil.validateEmailPattern(userRegistration.getEmail())) {
             errList.add("Invalid Email Address");
         }
         if (!ProjectUtil.validateContactPattern(userRegistration.getPhone())) {
             errList.add("Invalid Contact Number");
         }
+        if (StringUtils.isBlank(userRegistration.getPosition())) {
+            errList.add("Position is missing");
+        }
+        if (!userUtilityService.validatePosition(userRegistration.getPosition())) {
+            errList.add("Invalid Position");
+        }
```

Comment:
it should else block --> if position is not null then only you have to validate, right ?